doc/release/upcoming_changes/20722.expired.rst

```diff
@@ -0,0 +1,52 @@
+Changing to dtype of different size in F-contiguous arrays no longer permitted
+------------------------------------------------------------------------------
+
+Behavior deprecated in NumPy 1.11.0 allowed the following counterintuitive result::
+
+    >>> x = np.array(["aA", "bB", "cC", "dD", "eE", "fF"]).reshape(1, 2, 3).transpose()
+    >>> x.view('U1')  # deprecated behavior, shape (6, 2, 1)
+    DeprecationWarning: ...
+    array([[['a'],
+            ['d']],
+    
+           [['A'],
+            ['D']],
+    
+           [['b'],
+            ['e']],
+    
+           [['B'],
+            ['E']],
+    
+           [['c'],
+            ['f']],
+    
+           [['C'],
+            ['F']]], dtype='<U1')
+
+Now that the deprecation has expired, dtype reassignment only happens along the
+last axis, so the above will result in::
+
+    >>> x.view('U1')  # new behavior, shape (3, 2, 2)
+    array([[['a', 'A'],
+            ['d', 'D']],
+    
+           [['b', 'B'],
+            ['e', 'E']],
+    
+           [['c', 'C'],
+            ['f', 'F']]], dtype='<U1')
+
+When the last axis is not contiguous, an error is now raised in place of the `DeprecationWarning`::
+
+    >>> x = np.arange(3 * 4 * 5, dtype='<i4').reshape(3, 4, 5).transpose()
```

Comment:
Can you change this example to use strings too, so that the remarks below about `view("U1")` make sense?